src/pps/pps.proto

```diff
@@ -56,15 +56,17 @@ message Egress {
 
 message Job {
   string id = 1 [(gogoproto.customname) = "ID"];
+  Pipeline pipeline = 2;
```

Comment:
Really minor nit, but the pipeline should probably be the first field to match how we nest things in PFS.